language-extensions/python/README.md

```diff
@@ -7,19 +7,33 @@ For more information about SQL Server Language Extensions, refer to this [docume
 
 The Python extension version in this repository is compatible with SQL Server 2019 CU3 onwards. 
 
-Note that the Python Extension released in the current repository works with Python 3.7.x - to use the released package, follow [this tutorial](https://docs.microsoft.com/en-us/sql/machine-learning/install/custom-runtime-python?view=sql-server-ver15). For any other version of Python (3.6, 3.8, etc) you must modify and rebuild the Python Extension binaries using the following instructions.
+Note that the Python Extension released in the current repository works with Python 3.10.x - to use the released package, follow [this tutorial](https://docs.microsoft.com/en-us/sql/machine-learning/install/custom-runtime-python?view=sql-server-ver15). For any other version of Python (3.6, 3.8, etc) you must modify and rebuild the Python Extension binaries using the following instructions.
 
 ## Building
 
 ### Windows
+There are two alternatives to building this project.

```

Comment:
Need to write similar description for R: 

2 alternatives to building this project

Reply:
Also add the description to Java and R